.github/workflows/baremetal-regression-suite.yml

```diff
@@ -182,4 +182,4 @@ jobs:
         restore-keys: ${{ runner.os }}-m2
 
     - name: Build with Maven
-      run: mvn -B --activate-profiles jenkins-on-demand -Dmaven.gpg.skip=true -Dmaven.javadoc.skip=true -Dmaven.test.failure.ignore=false -Dhpccconn=http://eclwatch.default:8010 -Dwssqlconn=http://sql2ecl.default:8510 -DHPCC30117=open install
```

Comment:
what's the net affect of 2 active profiles?

Reply:
Good question, the two profiles are separate and won't interfere, jenkins-on-demand activates the remote tests and spark33 targets Spark 3.3 for the tests